core/templates/core/base.html

```diff
@@ -7,17 +7,19 @@
         <meta id="utmCookieDomain" value="{{ analytics.UTM_COOKIE_DOMAIN }}" />
         {% include 'core/google_tag_manager_head.html' %}
         <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
         <title>{% block head_title %}Welcome to great.gov.uk{% endblock %}</title>
 
+        {% block sharing_metadata %}{% endblock %}
```

Comment:
we can define default value in here, so we can do:

```
{% block sharing_metadata %}
    <meta property="og:image" content="{% static 'images/icons/opengraph-image.png' %}">
{% endblock %}
```

which ensures an image will be specified for all pages

Reply:
aha yes i forgot about that